tensorflow/models
# Update 4_Neural_Style_Transfer_with_Eager_Execution.ipynb

# Description
> I made changes for enabling the eager execution, so this code should work for users that are using Tensorflow version 2.x, rather than using the tensorflow 1.x.

## Type of change
- [ ] TensorFlow 2 migration

## Tests
> For the testing I used google colaboratory for testing the code and the result for it
1. First is that download the file or copy the code
2. Put the code or the file into google colaboratory
3. Change the runtime type into GPU used case
4. Make sure to use tensorflow version 2.x
5. Run All of the code cells

**Test Configuration**: You could use your own personal or local IDE or code editor for testing

## Checklist
> The image file, shows the result of the code
![result NST](https://user-images.githubusercontent.com/56186156/97677121-57918980-1ac4-11eb-97cc-171165cff3d4.JPG)
- [ ] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
